views/layout.erb

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" type="text/css" media="screen" href="/stylesheets/bulma.min.css" />
     <link rel="stylesheet" type="text/css" media="screen" href="/stylesheets/app.css" />
     <script type="text/javascript" src="/javascript/app.js"></script>
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
```

Comment:
Let's not add a font just yet and use the browser's default font, especially not some external JavaScript asset font. The whole app should be able to load and run without an internet connection.

Reply:
Oooh this is for the icons. For SVG icons just download the desired icon files and put them into `public/images/`. Previously I would just do a Google Images search for "SVG edit icon" and filter by Creative Commons. I recently discovered [iconduck.com](https://iconduck.com/search?query=edit+icon) which is a Free and Open Source collection of icons, and offers SVG downloads. We should probably just use individual icons from them.